src/main/java/it/gov/pagopa/payhub/activities/mapper/NotificationQueueMapper.java

```diff
@@ -0,0 +1,26 @@
+package it.gov.pagopa.payhub.activities.mapper;
+
+import it.gov.pagopa.payhub.activities.dto.debtposition.DebtPositionDTO;
+import it.gov.pagopa.payhub.activities.dto.debtposition.InstallmentDTO;
+import it.gov.pagopa.pu.p4paionotification.model.generated.NotificationQueueDTO;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Service;
+
+@Service
+@Lazy
+public class NotificationQueueMapper {
+
+    public NotificationQueueDTO mapDebtPositionDTO2NotificationQueueDTO(DebtPositionDTO debtPosition){
+        InstallmentDTO installment = debtPosition.getPaymentOptions().get(0).getInstallments().get(0);
```

Comment:
let's talk about this...
what should we notifiy about a debt position?
each installment?
the sum of the installment?

CC: @robertopasetti

Reply:
we have to send a message at debt position level, so the message can't contain info about: remittanceInfo, due date and amount. Remember (for the future) we have to manage multi-debtor destination, so for a single Debt Position IONotification has to send N message (where N is the number of different debtor in the DebtPosition)